### ISSUE TITLE
bug: `wmill sync push` fails

### ISSUE THREAD

**Issue by jrop** [windmill-labs/windmill#2228]
### Describe the bug

I was wanting to edit files locally, so I thought I would try a simple workflow:

```
❱ wmill sync pull --yes
❱ tree
.
└── u
    └── me
        ├── my_sandbox_script.bun.ts
        ├── my_sandbox_script.script.yaml
        ├── openai.resource.yaml
        ├── openai.variable.yaml
        ├── todo_notion.resource.yaml
        └── todo_notion.variable.yaml

2 directories, 6 files
❱ nvim u/me/my_sandbox_script.bun.ts
# make a simple edit: format the file
❱ wmill sync push --yes
You need to be up-to-date before pushing, pulling first.
Computing the files to update locally to match remote (taking .wmillignore into account)
remote (default) -> local: 0 changes to apply
Pull done, now pushing.


Computing the files to update on the remote to match local (taking .wmillignore into account)
remote (default) <- local: 1 changes to apply
~ script u/me/my_sandbox_script.bun.ts
import * as wmill from "windmill-client"
import * as wmill from "windmill-client";
import { Client as NotionClient } from "@notionhq/client";

type Notion = {
  token: string;
};
export async function main() {
  const database_id = '...';
  const auth = await wmill.getResource('u/me/todo_notion');
  const database_id = "...";
  const auth = await wmill.getResource("u/me/todo_notion");
  const client = new NotionClient({ auth: auth.token });
  return await client.databases.retrieve({
...
  });
}}

Applying changes to files ...
Creating script without parent u/me/my_sandbox_script.bun
Server failed. Bad Request: Sql error: error returned from database: new row for relation "script" violates check constraint "proper_id"
error: Uncaught (in promise) ApiError: Bad Request
        throw new ApiError(options, result, error);
              ^
    at catchErrorCodes (https://deno.land/x/windmill@v1.95.1/windmill-api/core/request.ts:265:15)
    at https://deno.land/x/windmill@v1.95.1/windmill-api/core/request.ts:301:17
    at eventLoopTick (ext:core/01_core.js:183:11)
```

I was surprised to see a SQL constraint from such a simple edit.

### To reproduce

See above

### Expected behavior

I would have expected the sync command to push the changes to the file.

### Browser information

Version 116.0.5845.96 (Official Build) (64-bit)

### Application version

Windmill CE v1.166.0-7-g76edfd6d Update: v1.166.0-7-g76edfd6d -> v1.166.1  

### Additional Context

wmill v1.166.1

**Comment by jrop** [comment]
(obligatory: this is a cool project: please don't read a disgruntled tone into the bug report :smile:)

**Comment by rubenfiszel** [comment]
Actually the SQL constraint is good, the CLI is mistakenly thinking bun is part of the path to push. It's a simple fix, I will do it first thing tomorrow. Cheers

**Comment by rubenfiszel** [comment]
Actually I did a quick fix on 1.167.0 if you want to give it a try (should be deployed in a few mins), just upgrade with `wmill upgrade`

**Comment by jrop** [comment]
That worked, thank you!
